domador.js

```diff
@@ -490,7 +490,11 @@ Domador.prototype.process = function process (el) {
     case 'PRE':
       if (this.options.fencing) {
         this.append('\n\n');
-        this.output('```' + (this.options.fencinglanguage(el) || '') + '\n');
+        var fencinglanguage = this.options.fencinglanguage(el);
```

Comment:
Cool, say, everything between 491-499 could become `domador.languagefence()`, or what would be your preferred function name?